app/views/examples/wizard/where-do-you-live.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "layouts/default.njk" %}
+
+{% set title = "Where do you live?" %}
+
+{% block pageNavigation %}
+  {{ govukBackLink({
+    href: paths.back
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form method="post" action="{{ paths.current }}" novalidate>
```

Comment:
You didn't want to include the form layout which abstracts away the paths logic?

Reply:
I’m wondering if such layouts should be a different feature (I’ve created page, question and check answer layouts on our current prototype). The important thing the wizard provides is `path.*` values. How you use them is up to you.

Reply:
Yeah, but if the form doesn't post back to itself the fork logic won't execute

Reply:
Not sure I understand. The examples include the forms posting to `paths.current` (arguably more visible in a template than an abstracted layout), and even if you don’t do this, isn’t the default behaviour to post to the current page anyway?